src/server/command/worldedit/pattern.zig

```diff
@@ -0,0 +1,60 @@
+const std = @import("std");
+
+const main = @import("root");
+const AliasTable = main.utils.AliasTable;
+const Block = main.blocks.Block;
+const ListUnmanaged = main.ListUnmanaged;
+const NeverFailingAllocator = main.heap.NeverFailingAllocator;
+
+blocks: AliasTable(Entry),
```

Comment:
First I have implemented it as a struct within a file, but it was only thing here, so I figured it doesn't make sense to have a single struct within a file of the same name.